Resolve syncing conflicts from live-sync-work to live

openpublishbuild opened the pull request:
Pull request opened by OPS to resolve syncing conflicts from live-sync-work to live.

Commit:
Consistently use braces for single line blocks

In some cases you where using {} for single line statements following conditionals such as if.  In other cases they where omitted.  As per StyleCop guidelines the usage should be consistent, and using {} is recommended.

Commit:
Removed RC from Visual Studio 2017 product name

Commit:
Merge pull request #1421 from rpetrusha/csproj

Removed RC from Visual Studio 2017 product name

Commit:
Redirects for Visual Studio 2015 tutorials (#1423)

* Redirects for Visual Studio 2015 tutorials

* fixed broken link

* Changed 'updated' to 'retired'

alexwiese committed:
Update index.md (#1430)

mairaw committed:
Update .openpublishing.publish.config.json

saldana committed:
Merge pull request #1431 from dotnet/pdf

Update .openpublishing.publish.config.json

mairaw committed:
fixed broken link (#1432)

mairaw committed:
added links between versions (#1424)

* added links between versions

* updating links to nuget and asp.net

* addressed feedback

* fixed broken link

mairaw committed:
Merge pull request #1440 from dotnet/master

Update live with current master

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C1%5C23%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201701231635231664-2%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

bleroy committed:
Small tweaks to the full VS 2017 Core tutorial

mairaw committed:
updated title (#1443)

BillWagner committed:
Reorganize the C# Guide TOC (#1318)

* Use a sub-TOC for the Tour

This is a first experiment with the sub TOC. If I like the way this
works, I’ll use it throughout the C# Guide

* Update GettingStarted, Tutorials

* Update C# Interactive TOC

* move language reference.

* keywords and operators TOC updates.

* update preprocessor section

* update compiler options

* fix OPS build errors

* finish the language reference.

* top level C# programming guide

This needs to be broken down much more in the lower levels.

* update inside a C# program.

* use sub-toc for arrays.

* use sub tor for classes and structs

* update the delegates section

* update events

* update exceptions TOC

* post-merge updates

* update the file system section.

* update the generics section

* update from build errors

* update the interop section

* update file names.

* update section on main

* update XML doc section.

* revert incorrect file name changes.

* update strings and types.

* update concepts

* fix a global replace error

* update assemblies and gas

* update concepts async section

* fix build errors from global search / replace

* co-contra variance, expression trees

* update the linq topics.

* all sub TOCs are in place.

* update sub-toc after rebase.

* fix a rename from a merge conflict.

* fix a build error.

* one final build warning...

mairaw committed:
Merge branch 'csproj' into master

Commit:
Add braces to single-line statement in line 16

mairaw committed:
reverting merge (#1447)

Commit:
Getting Started with F# in Visual Studio Code with Ionide: add visual aid for an often missed step (#1457)

* "Create new directory" visual aid

* Added "open folder" visual aid to article

IEvangelist committed:
Minor correction in the opening sentence on Tuples (#1458)

The first sentence needed to be corrected.

mairaw committed:
fixed typos (#1459)

mairaw committed:
preparing changes for RC3 release

a-ctor committed:
Added a missing bracket in a code example

Commit:
Merge pull request #1404 from quintusm/patch-2

Consistently use braces for single line blocks

Commit:
Merge pull request #1465 from a-ctor/patch-1

Added a missing bracket in a code example

TingluoHuang committed:
fix dotnet test --nobuild option, should be --no-build (#1466)

* fix --nobuild option, should be --no-build

* Update dotnet-test.md

mairaw committed:
Merge pull request #1473 from mairaw/rc3

changing tooling version name to RC3

mairaw committed:
readding deleted files (#1461)

mairaw committed:
removing caps link (#1463)

* removing caps link

* addressed feedback

mairaw committed:
Merge pull request #1478 from dotnet/master

Update live with current master

openpublishtest commented:
### :x: Validation status: errors


For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C1%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702010022130668-2%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

herohua committed:
resolve syncing conflicts

openpublishtest commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/visual-basic/language-reference/error-messages/identifier-expected.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/language-reference/error-messages/identifier-expected.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/error-messages/identifier-expected?branch=pr-en-us-2) |
[docs/fsharp/tutorials/getting-started/getting-started-vscode.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/fsharp/tutorials/getting-started/getting-started-vscode.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/fsharp/tutorials/getting-started/getting-started-vscode?branch=pr-en-us-2) |
[docs/samples-and-tutorials/index.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/samples-and-tutorials/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/samples-and-tutorials/index?branch=pr-en-us-2) |
[docs/standard/managed-code.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/standard/managed-code.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/standard/managed-code?branch=pr-en-us-2) |
[docs/standard/native-interop.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/standard/native-interop.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/standard/native-interop?branch=pr-en-us-2) |
[docs/visual-basic/programming-guide/language-features/procedures/extension-methods.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/programming-guide/language-features/procedures/extension-methods.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/programming-guide/language-features/procedures/extension-methods?branch=pr-en-us-2) |
[docs/visual-basic/reference/command-line-compiler/subsystemversion.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/reference/command-line-compiler/subsystemversion.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/reference/command-line-compiler/subsystemversion?branch=pr-en-us-2) |
[index.md](https://github.com/openpublishtest/docs/blob/live-sync-work/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/index?branch=pr-en-us-2) |
[docs/visual-basic/programming-guide/concepts/index.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/programming-guide/concepts/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/programming-guide/concepts/index?branch=pr-en-us-2) |
[docs/visual-basic/programming-guide/concepts/attributes/index.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/programming-guide/concepts/attributes/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/programming-guide/concepts/attributes/index?branch=pr-en-us-2) |
[docs/visual-basic/language-reference/modifiers/assembly.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/language-reference/modifiers/assembly.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/modifiers/assembly?branch=pr-en-us-2) |
[docs/visual-basic/language-reference/modifiers/module-keyword.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/language-reference/modifiers/module-keyword.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/modifiers/module-keyword?branch=pr-en-us-2) |
[docs/visual-basic/language-reference/statements/attribute-list.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/language-reference/statements/attribute-list.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/statements/attribute-list?branch=pr-en-us-2) |
[docs/visual-basic/language-reference/statements/parameter-list.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/language-reference/statements/parameter-list.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/statements/parameter-list?branch=pr-en-us-2) |
[docs/visual-basic/language-reference/attributes.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/language-reference/attributes.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/attributes?branch=pr-en-us-2) |
[docs/visual-basic/language-reference/error-messages/attribute-cannot-be-applied-because-the-format-of-the-guid-is-not-correct.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/language-reference/error-messages/attribute-cannot-be-applied-because-the-format-of-the-guid-is-not-correct.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/error-messages/attribute-cannot-be-applied-because-the-format-of-the-guid-is-not-correct?branch=pr-en-us-2) |
[docs/visual-basic/language-reference/error-messages/elementname-is-obsolete-visual-basic-warning.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/language-reference/error-messages/elementname-is-obsolete-visual-basic-warning.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/language-reference/error-messages/elementname-is-obsolete-visual-basic-warning?branch=pr-en-us-2) |
[docs/csharp/misc/cs1558.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/misc/cs1558.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs1558?branch=pr-en-us-2) |
[docs/csharp/misc/cs1559.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/misc/cs1559.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs1559?branch=pr-en-us-2) |
[docs/csharp/programming-guide/statements-expressions-operators/lambda-expressions.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/statements-expressions-operators/lambda-expressions.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/statements-expressions-operators/lambda-expressions?branch=pr-en-us-2) |
[docs/csharp/programming-guide/statements-expressions-operators/statements.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/statements-expressions-operators/statements.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/statements-expressions-operators/statements?branch=pr-en-us-2) |
[docs/csharp/programming-guide/unsafe-code-pointers/fixed-size-buffers.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/unsafe-code-pointers/fixed-size-buffers.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/unsafe-code-pointers/fixed-size-buffers?branch=pr-en-us-2) |
[docs/csharp/programming-guide/xmldoc/exception.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/xmldoc/exception.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/xmldoc/exception?branch=pr-en-us-2) |
[docs/csharp/misc/cs5001.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/misc/cs5001.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/misc/cs5001?branch=pr-en-us-2) |
[docs/csharp/programming-guide/index.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/index?branch=pr-en-us-2) |
[docs/csharp/programming-guide/classes-and-structs/methods.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/classes-and-structs/methods.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/classes-and-structs/methods?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/index.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/index?branch=pr-en-us-2) |
[docs/visual-basic/programming-guide/language-features/statements.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/visual-basic/programming-guide/language-features/statements.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/visual-basic/programming-guide/language-features/statements?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/async/async-return-types.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/async/async-return-types.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/async-return-types?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/async/asynchronous-programming-with-async-and-await.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/async/asynchronous-programming-with-async-and-await.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/asynchronous-programming-with-async-and-await?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/async/cancel-async-tasks-after-a-period-of-time.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/async/cancel-async-tasks-after-a-period-of-time.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/cancel-async-tasks-after-a-period-of-time?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/async/cancel-remaining-async-tasks-after-one-is-complete.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/async/cancel-remaining-async-tasks-after-one-is-complete.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/cancel-remaining-async-tasks-after-one-is-complete?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/async/control-flow-in-async-programs.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/async/control-flow-in-async-programs.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/control-flow-in-async-programs?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/async/fine-tuning-your-async-application.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/async/fine-tuning-your-async-application.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/fine-tuning-your-async-application?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/handling-reentrancy-in-async-apps?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/async/how-to-make-multiple-web-requests-in-parallel-by-using-async-and-await.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/async/how-to-make-multiple-web-requests-in-parallel-by-using-async-and-await.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/how-to-make-multiple-web-requests-in-parallel-by-using-async-and-await?branch=pr-en-us-2) |
[docs/csharp/programming-guide/concepts/async/index.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/concepts/async/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/concepts/async/index?branch=pr-en-us-2) |
[docs/csharp/programming-guide/exceptions/compiler-generated-exceptions.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/exceptions/compiler-generated-exceptions.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/compiler-generated-exceptions?branch=pr-en-us-2) |
[docs/csharp/programming-guide/exceptions/creating-and-throwing-exceptions.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/exceptions/creating-and-throwing-exceptions.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/creating-and-throwing-exceptions?branch=pr-en-us-2) |
[docs/csharp/programming-guide/exceptions/exception-handling.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/exceptions/exception-handling.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/exception-handling?branch=pr-en-us-2) |
[docs/csharp/programming-guide/exceptions/exceptions-and-exception-handling.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/exceptions/exceptions-and-exception-handling.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/exceptions-and-exception-handling?branch=pr-en-us-2) |
[docs/csharp/programming-guide/exceptions/how-to-catch-a-non-cls-exception.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/exceptions/how-to-catch-a-non-cls-exception.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/how-to-catch-a-non-cls-exception?branch=pr-en-us-2) |
[docs/csharp/programming-guide/exceptions/how-to-execute-cleanup-code-using-finally.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/exceptions/how-to-execute-cleanup-code-using-finally.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/how-to-execute-cleanup-code-using-finally?branch=pr-en-us-2) |
[docs/csharp/programming-guide/exceptions/how-to-handle-an-exception-using-try-catch.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/exceptions/how-to-handle-an-exception-using-try-catch.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/how-to-handle-an-exception-using-try-catch?branch=pr-en-us-2) |
[docs/csharp/programming-guide/exceptions/index.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/exceptions/index.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/index?branch=pr-en-us-2) |
[docs/csharp/programming-guide/exceptions/using-exceptions.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/exceptions/using-exceptions.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/exceptions/using-exceptions?branch=pr-en-us-2) |
[docs/csharp/programming-guide/file-system/how-to-copy-delete-and-move-files-and-folders.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/file-system/how-to-copy-delete-and-move-files-and-folders.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-copy-delete-and-move-files-and-folders?branch=pr-en-us-2) |
[docs/csharp/programming-guide/file-system/how-to-create-a-file-or-folder.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/file-system/how-to-create-a-file-or-folder.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-create-a-file-or-folder?branch=pr-en-us-2) |
[docs/csharp/programming-guide/file-system/how-to-create-a-key-in-the-registry.md](https://github.com/openpublishtest/docs/blob/live-sync-work/docs/csharp/programming-guide/file-system/how-to-create-a-key-in-the-registry.md) | :white_check_mark:Succeeded | [View](https://ppe.docs.microsoft.com/en-us/optest/core-docs/articles/csharp/programming-guide/file-system/how-to-create-a-key-in-the-registry?branch=pr-en-us-2) |

This comment lists only the first 50 files in the pull request.
For more details, please refer to the [build report](https://opbuildstoragesandbox2.blob.core.windows.net/report/2017%5C2%5C6%5C24d48890-b34f-1f68-1499-009fa4ca25b2%5CPullRequest%5C201702060336143624-2%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
